lib/srv/discovery/discovery.go

```diff
@@ -466,6 +472,75 @@ func (s *Server) Wait() error {
 	return nil
 }
 
+func (s *Server) watchCARotations(ctx context.Context) error {
+	clustername, err := s.Auth.GetClusterName()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	watcher, err := s.Auth.NewWatcher(ctx, types.Watch{
```

Comment:
Pretty sure you can watch from `s.AccessPoint`, the discovery access point has all the CAs cached already.